sites/all/modules/custom/training/training.pages.inc

```diff
@@ -0,0 +1,249 @@
+<?php
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu']
+ *
+ * @return string
+ */
+function training_display_page(){
+  return t('This is the landing page of the Training module');
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu/custom_access']
+ *
+ * @return string
+ */
+function training_access_page(){
+  return t('This is page to hook_permission (custom access training)');
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu/permissioned']
+ *
+ * @return string
+ */
+function training_permissioned_page(){
+  return t('This is page to hook_permission (permissioned access training)');
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu/callback']
+ *
+ * @return string
+ */
+function training_callback_page(){
+  return t('The menu entry for this page is of type MENU_CALLBACK');
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu/tabs'], $menu['menu/tabs/default']
+ *
+ * @return string
+ */
+function training_tab_page(){
+  return t('This is the "tabs" menu entry');
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from all the $menu['menu/tabs/*'], exept $menu['menu/tabs/default']
+ *
+ * @param $tnum
+ *  The tab number
+ * @param int $level
+ *  Level of tag: menu ($level=0) of menu/default ($level=1)
+ * @return string
+ */
+function training_tab_num_page($tnum, $level = 0){
+  $leveltest = $level == 0 ? '"basic tabs"' : '"default tabs"';
+  $res = 'This is the tab "' . $tnum . '" in the ' . $leveltest . 'example';
+  return $res;
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu/arguments/%/%']
+ *
+ * @param $one
+ * @param $two
+ * @return string
+ */
+function training_arguments_page ($one, $two) {
+  return 'Argument 1 = <em>' . $one . '<em><br>' .
+    'Argument 2 = <em>' . $two . '<em>';
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu/placeholder/%/display']
+ *
+ * @param $smth
+ *  Any value user enters instead of % in menu/placeholder/%/display
+ * @return mixed
+ */
+function training_placeholder_page($smth){
+  return $smth;
+}
+
+/**
+ * Page callback for training_menu(),
+ * called from $menu['menu/default_arg/%']
+ *
+ * @param $arg
+ *  Any value user enters instead of % in menu/default_arg/%
+ * @return string
+ */
+function training_default_arg_page($arg){
+  switch ($arg) {
+    case 1:
+      return 'one';
```

Comment:
Add variable. In case construction assign value to variable. In the end odf the function write return $variable.